pwa/src/templates/sidebar/Sidebar.module.css

```diff
@@ -4,17 +4,17 @@
   justify-content: space-between;
 
   background: white;
```

Comment:
This can be `backround-color` instead of just `background`. Also, we can use a token here.